src/bors/command/mod.rs

```diff
@@ -22,6 +22,8 @@ pub enum BorsCommand {
     Try {
         /// Parent commit which should be used as the merge base.
         parent: Option<Parent>,
+        /// The CI workflow to run.
+        jobs: Option<Vec<String>>,
```

Comment:
```suggestion
        jobs: Vec<String>,
```
This should be easier. Passing an empty `jobs=` list should be an error.